apps/taskman/tests/test_flaw_model_integration.py

```diff
@@ -0,0 +1,132 @@
+"""
+    Integration tests to validate Taskman REST API endpoints
+"""
+from datetime import datetime
+
+import pytest
+from django.utils.timezone import make_aware
+
+import apps.taskman.mixins as mixins
+import osidb.models as models
+from apps.taskman.exceptions import MissingJiraTokenException
+from apps.taskman.service import JiraTaskmanQuerier
+from osidb.models import Flaw, FlawSource, FlawType, Impact
+from osidb.tests.factories import AffectFactory, FlawFactory
+
+pytestmark = pytest.mark.integration
+
+
+class TestFlawModelIntegration(object):
+    def test_not_syncing(
+        self, monkeypatch, acl_read, acl_write, user_token, auth_client, test_api_uri
+    ):
+        """
+        Test that JIRA_TASKMAN_AUTO_SYNC_FLAW=0 allows user to save/create without token.
+        """
+        sync_count = 0
+
+        def mock_create_or_update_task(self, flaw, fail_if_exists=False):
+            nonlocal sync_count
+            sync_count += 1
+
+        monkeypatch.setattr(
+            JiraTaskmanQuerier, "create_or_update_task", mock_create_or_update_task
+        )
+        monkeypatch.setenv("JIRA_TASKMAN_AUTO_SYNC_FLAW", 0)
```

Comment:
this test should work if the variable is simply unset, so no need to monkeypatch